.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -52,8 +51,8 @@ jobs:
           install.packages("devtools", dependencies=TRUE)
           devtools::install_deps()
         shell: Rscript {0}
-        env: # Set individual GH PAT
-          GITHUB_PAT: ${{ secrets.PAT_FOR_DEVTOOLS }}
+        env:
+          GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why is it called `GITHUB_PAT` here instead of `GITHUB_TOKEN`?

Reply:
The R install process uses the environment variable `GITHUB_PAT` to interact with Github. 
Note that we install the R packages here directly within the Github action, while in the other places we build the docker image, which forward the `GITHUB_TOKEN` to the docker build image script (which then transforms `GITHUB_TOKEN` to `GITHUB_PAT`)

Reply:
OK.